spec/rails_helper.rb

```diff
@@ -1,6 +1,6 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 require 'spec_helper'
-ENV['RAILS_ENV'] ||= 'test'
+ENV['RAILS_ENV'] = 'test'
```

Comment:
why get rid of the ||= for environment checks?

Reply:
Because if ENV['RAILS_ENV'] == 'DEVELOPMENT' it's not changing to 'TEST'